app/javascript/components/util/InfoTooltip.js

```diff
@@ -14,6 +14,20 @@ const TOOLTIP_TEXT = {
 
   lastDateOfExposure: <div> Used by the system to automatically calculate the monitoring period. </div>,
 
+  primaryLanguage: (
+    <div>
+      <i>Primary Language</i> is used to determine the translations for what the monitoree sees/hears. If a language is not fully supported, a warning to users
+      will appear.
+    </div>
+  ),
+
+  secondaryLanguage: (
+    <div>
+      <i>Secondary Language</i> is not used to determine which language the system sends messages to the monitoree in. Information in this field can be used to
```

Comment:
Great idea putting tooltips here. I'm a bit confused by the wording on this one. The first line surprised me by being a negative. How exactly do we use the secondary language? It's not super clear to me from this

Reply:
Kelly gave me the text for both of these tooltips, so that would be a question for her or @holmesie